File: app/src/test/java/vassar/cmpu203/vassardiningapp/DietaryRestrictionTest.java
package vassar.cmpu203.vassardiningapp;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertThrows;

import org.junit.jupiter.api.Test;

import vassar.cmpu203.vassardiningapp.model.DietaryRestriction;

public class DietaryRestrictionTest {

    @Test
    public void testGetById() {
        assertEquals(DietaryRestriction.getById("1"), DietaryRestriction.VEGETARIAN);
        assertThrows(RuntimeException.class, ()->DietaryRestriction.getById("999"));
    }
}

File: app/src/test/java/vassar/cmpu203/vassardiningapp/DietaryRestrictionTest.kt
package vassar.cmpu203.vassardiningapp

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import vassar.cmpu203.vassardiningapp.model.DietaryRestriction
import vassar.cmpu203.vassardiningapp.model.DietaryRestriction.Companion.getById

class DietaryRestrictionTest {
    @Test
    fun testGetById() {
        Assertions.assertEquals(getById("1"), DietaryRestriction.VEGETARIAN)
        Assertions.assertThrows(RuntimeException::class.java) { getById("999") }
    }
}